scripts/missions/amk/helpers.lua

```diff
@@ -57,11 +57,7 @@ end
 -- a local var that is set once by the first player is called by onMobDeath
 -- The rest of the players in alliance get the same outcome as the first
 xi.amk.helpers.cardianOrbDrop = function(mob, player, orb)
```

Comment:
What are the conditions for cardians to drop this key item? Can the item drop forever if you've completed AMK? Or do they only drop during this mission and if you want repeat KIs then you have to buy them from squintox?

Either way, your most recent commit removed the mission check all together

Reply:
![image](https://github.com/LandSandBoat/server/assets/85970456/e86aaa8e-0b89-4c89-bdaa-01d6c5529308)

I can't, however, find any information about whether or not the key items can drop before flagging mission 6 from shantotto.

Reply:
Easy to go get caps on retail

Reply:
For now, this condition seems fine: `player:getCurrentMission(xi.mission.log_id.AMK) < xi.mission.id.amk.AN_ERRAND_THE_PROFESSORS_PRICE`

Reply:
I made a request in FFXI Captures discord. I don't have a retail account personally.